app/src/main/kotlin/com/wire/android/ui/home/sync/FeatureFlagNotificationViewModel.kt

```diff
@@ -70,30 +59,40 @@ class FeatureFlagNotificationViewModel @Inject constructor(
      * state.
      */
     fun loadInitialSync() {
-        viewModelScope.launch {
-            currentSessionUseCase().let { currentSessionResult ->
-                when (currentSessionResult) {
-                    is CurrentSessionResult.Failure -> {
-                        appLogger.e("Failure while getting current session from FeatureFlagNotificationViewModel")
-                    }
+        viewModelScope.launch { initialSync() }
+    }
+
+    suspend fun initialSync() {
```

Comment:
seems like this is only being used inside `loadInitialSync()` so maybe it could be private?

Reply:
no, it's also used in tests :)